azure/v12.1.0/README.md

```diff
@@ -0,0 +1,11 @@
+## :zap:  Giant Swarm Release 12.1.0 for Azure :zap:
+
+**If you are upgrading from 12.0.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades all helm releases to helm 3.
+
```

Comment:
```suggestion

This lets us benefit from the improved security and keep up to date with the community. We also removed the Tiller deployment from tenant clusters, removing its gRPC endpoint, which reduces operational complexity. 

```

@rossf7 a while back we drafted release notes for this. Still seems relevant. WDYT? https://www.dropbox.com/scl/fi/rcle9tj1oj54netb5sqkk/Helm-3-Release-Notes.paper?dl=0&rlkey=nq013qe7hn8r0lh7iasys1mm1

Reply:
```suggestion
This lets us benefit from the improved security model and keep up to date with the community. We also remove the Tiller deployment from the giantswarm namespace, removing its gRPC endpoint, which reduces operational complexity.

If you are still using Helm 2 then these Helm releases will not be affected. However we encourage you to upgrade to Helm 3. As Helm 2 support ends on November 13th 2020. https://helm.sh/blog/helm-v2-deprecation-timeline/
```

Yes I think we should add more detail here. WDYT?

Reply:
Nice, I will put it into the changelog.